deploy/crds/push_v1alpha1_unifiedpushserver_crd.yaml

```diff
@@ -69,6 +69,15 @@ spec:
                 - backendSecretName
                 type: object
               type: array
+            oAuthResourceRequirements:
```

Comment:
Do we need to bump any version of the crd now that there is a change to it? cc @grdryn

Reply:
I don't think so, given that we're only introducing optional new stuff. The existing API will still work as expected -- nothing has changed or been removed there.